### PR TITLE
Error bars for BarChart

### PR DESCRIPTION
**Pull request by MaximeBouton**
- implement error bars for barchart
- add an example to the doc 

*Note*: the way I implemented it might be clumsy, there is a lot of repeated code with the implementation for the `Linear` type. Any suggestions are welcome.

### CODE REVIEW

File: src/PGFPlots.jl [hunk 1]
```diff
@@ -431,16 +434,33 @@ function plotHelperErrorBars(o::IO, p::Linear)
     println(o, ", error bars/.cd, ")
     optionHelper(o, errorbarsMap, p.errorBars, otherText=["x dir=both", "x explicit", "y dir=both", "y explicit"])
     println(o, "]")
-    x = vcat(p.data, p.errorBars.data)
     println(o, "table [")
     println(o, "x error plus=ex+, x error minus=ex-, y error plus=ey+, y error minus=ey-")
     println(o, "] {")
     println(o, "x y ex+ ex- ey+ ey-")
+    x = vcat(p.data, p.errorBars.data)
```
Comment (mykelk) [inline src/PGFPlots.jl]: Why did this line need to be moved? It looks like it came before the table originally, right?

Comment (MaximeBouton) [reply src/PGFPlots.jl]: Sorry it was unnecessary, I moved it while I was prototyping and forgot to move it back. I will revert this change

**Comment by mykelk** [comment]
Cool. You can merge once the travis finishes. Please also tag a release for attobot. Thanks!

**Comment by MaximeBouton** [comment]
I don't have the rights to merge or tag

**Comment by mykelk** [comment]
You should have access now.

**Pull request merged**